doc/authentication-backends/HTTP-authentication-module.md

```diff
@@ -112,6 +112,15 @@ A body can be missing in the first data chunk read from a socket, leading to str
     * 200, `true` or `false` in body
     * anything else - will be treated as `false`
 
+### Method `get_certs`
+
+* **Description** Provides a list of certificates corresponding to a given user. Works like third party Certificate Authority service. The certificates must be given in `PEM` format.
```

Comment:
**Description** Provides a list of certificates corresponding to a given user. Works like a third party Certificate Authority service. The certificates must be given in a `PEM` format.